app/_components/commodities/components/GridHeadingsMobile.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+
+import { FaArrowDown, FaArrowsUpDown, FaArrowUp } from 'react-icons/fa6';
+import { Button, GridItem, SimpleGrid } from '@chakra-ui/react';
+import selectColor from '@/app/_hooks/fontColorSelector';
+import useColorMode from '@/app/_hooks/useColorMode';
+
+interface IGridHeadingsProps {
+  filter: string;
+  isBuying: boolean;
+  setFilter: React.Dispatch<React.SetStateAction<any>>;
+  ascending: boolean;
+  setAscending: React.Dispatch<React.SetStateAction<boolean>>;
+}
+
+const GridHeadingsMobile: React.FC<IGridHeadingsProps> = ({
+  filter,
+  isBuying,
+  setFilter,
+  ascending,
+  setAscending,
+}) => {
+  const { isDark } = useColorMode();
+
+  const gridHeadings: {
+    id: number;
+    text: string;
+    sortByString: string | null;
+  }[] = [
+    { id: 0, text: 'Sell Price', sortByString: 'sellPrice' },
+    { id: 1, text: 'Buy Price', sortByString: 'buyPrice' },
+    { id: 2, text: 'Supply', sortByString: 'supply' },
+    { id: 3, text: 'Demand', sortByString: 'demand' },
+    { id: 4, text: 'System', sortByString: null },
+    { id: 5, text: 'Distance', sortByString: 'distance' },
+  ];
+
+  const unnecessaryHeadings = (newFilter: string | null) => {
+    if (isBuying && newFilter === 'buyPrice') return true;
+    if (isBuying && newFilter === 'demand') return true;
+    if (!isBuying && newFilter === 'sellPrice') return true;
+    if (!isBuying && newFilter === 'supply') return true;
+    return false;
+  };
+
+  const renderArrowIcon = (newFilter: string) => {
+    if (filter === newFilter) {
+      return ascending ? <FaArrowDown /> : <FaArrowUp />;
+    }
+    return <FaArrowsUpDown />;
+  };
+
+  return (
+    <SimpleGrid
+      columns={[4, 4]}
+      width="100%"
+      fontWeight="bold"
+      borderBottom="1px"
+      borderColor={selectColor(isDark, 'border')}
+      rowGap={1}
+    >
+      {gridHeadings.map((heading) => {
+        if (unnecessaryHeadings(heading.sortByString)) return null;
```

Comment:
At a low mobile width you can see the 4 headings, but only 3 cells worth of content. Also as above the content alignment is out of whack with the header cell alignment

![image](https://github.com/ed-pilots-network/frontend/assets/319883/fa970c06-3c03-41a9-9e32-0cdd3578dcaf)

Reply:
fixed:

![Screenshot 2023-08-31 at 10-17-43 EDPN - Commodities](https://github.com/ed-pilots-network/frontend/assets/67390534/106cbed0-4a93-460d-b3b8-f9efa30e42cb)